plenum/server/node.py

```diff
@@ -3225,7 +3226,8 @@ def _process_new_view_accepted(self, msg: NewViewAccepted):
         for i in self.replicas.keys():
             if i != MASTER_REPLICA_INDEX:
                 self.replicas.send_to_internal_bus(msg, i)
-            self.primary_selected(i)
+            else:
+                self.primary_selected(i)
```

Comment:
Why this change?

Reply:
Because now only master instance selected the Primary, backup instances haven't selected them yet. Please note that this is the place when we received NEW_VIEW, re-ordering is not started and finished yet.